tests/data/test_particle.py

```diff
@@ -20,7 +20,7 @@
             pid=443,
             mass=3.0969,
             width=9.29e-05,
-            state=ParticleQuantumState(spin=1, charge=0),
+            state=QuantumState[float](spin=1, charge=0),
```

Comment:
We may want to add a `__repr__` check on `ComplexEnergy` as well, then that typo would have been fished out. Or even better: somehow automate this test by running over all classes in the `data` module that have a `__repr__` definition (not sure if that's possible).

Reply:
see 6511864073ed8b79407c616fe1c6d4cf97552760